.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: node_js
 node_js:
+  - "stable"
+  - 11
+  - 10
   - 9
   - 8
+install:
+  - yarn install
+script:
+  # - yarn run lint
```

Comment:
I suggest we check for code formatting too using eslint, to have consistency. Will have to set up a lint script in `package.json`. It will start giving some errors, will have to fix them too.

Reply:
Yep agree on code formatting. I believe the rules are already set in `.eslintrc`. Will Travis run this automatically for every commit?

Reply:
Yes once in scripts section of `package.json` we put something like:

    "lint": "eslint *.js",

Since we are converting stuff to ts, maybe wait of #9 to get merged.

Reply:
On second thought maybe we need two commands `lint_js` and `lint_ts` because there will still be js code in example directory.
Also once we run lint the examples directory there will be errors. Is it ok to fix them in this PR, or should we do another PR for that.

Reply:
It's ok to fix them in this PR. Thanks again for your input @vaibhavhrt I'm learning a lot from it!

Reply:
OK, I will get to it. You can check the travis test details here for now: https://travis-ci.com/vaibhavhrt/react-physics-dragger
Once this PR is merged you will see tests on your travis account and also on every github PR, commits too.